src/components/Queue.vue

```diff
@@ -90,12 +91,7 @@ export default {
     },
     computed: {
         currentlyShown: function() {
-            return (
-                this.fail
-                || this.validate
-                || (this.vouchers.length >= 1 && !this.netMgr.online)
-                || this.queue.sendingStatus
-            );
+            return (this.fail || this.validate || (this.vouchers.length >= 1 && !this.netMgr.online) || this.queue.sendingStatus);
```

Comment:
I think split this back to multiline so we can see more clearly what is going on.